tests/behat/features/media.feature

```diff
@@ -0,0 +1,8 @@
+@d7
+Feature: Check that MediaTrait works
+
+  @api
+  Scenario: Assert that user can attach file to media field
```

Comment:
please use `Scenario: Assert 'When I attach the file :file to :field_name media field`